src/query.ts

```diff
@@ -507,6 +510,45 @@ export class Query<T extends Model> {
     return _.pick(instance, Object.keys(getModelAssociations(this.model))) as T;
   }
 
+/**
+ * Returns the associations that correlate to the current include options
+ * @param {boolean} required? Whether or not to only get required associations (associations that are not nullable).
+ * @return {any[]} An array of association descriptors.
+ */
+  private getIncludedAssociations(required?: boolean): any[] {
```

Comment:
* Ah, I left it in there because I was going to use it in a different PR that overhauls storage of associations.
* Unfortunately there is no type available in the sequelize definition for this object. I can add one locally.